.github/continuous-integration.yml

```diff
@@ -0,0 +1,40 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
```

Comment:
If you prefer not to quote strings in YAML files, let me know, then I will adjust.